test/reviewsTests/reviewBreakdown.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+const {default: userEvent} = require('@testing-library/user-event');
+import data from '../../client/src/components/reviews/apiExample.js';
+
+import {Characteristics} from '../../client/src/components/reviews/CharacteristicsInputs.jsx';
+import {ReviewButtons} from '../../client/src/components/reviews/reviewListComponents/AdditionalButtons.jsx';
+import Reviews from '../../client/src/components/reviews/ReviewsMain.jsx';
+
+// CHARACTERISTICS
+describe('Characteristics for NewReview', () => {
```

Comment:
here is a useful stack overflow page on it vs test and how they should be used: https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest. It's all about readability but. I think you could add a "should" in the title of your first it statement.